samples/hyperparameter_tuning/my-hyperparameter-job.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: HyperParameterTuningJob
+metadata:
+  name: my-hyperparameter-job
+spec:
+  hyperParameterTuningJobName: my-hyperparamater-job
+  hyperParameterTuningJobConfig:
+    strategy: Bayesian
+    hyperParameterTuningJobObjective:
```

Comment:
Both of these needs comments to `# Modify these parameters to meet your own script's needs`